packages/eas-json/src/types.ts

```diff
@@ -21,7 +21,7 @@ export interface EasJson {
     version?: string;
     requireCommit?: boolean;
     appVersionSource?: AppVersionSource;
-    promptToConfigurePushNotfications?: boolean;
+    promptToConfigurePushNotifications?: boolean;
```

Comment:
If this typo is required to stay to maintain compatibility with another package/library, let me know and I can pull this change